CMakeLists.txt

```diff
@@ -202,7 +202,11 @@ endif( )
 
 # Package specific CPACK vars
 if(HIP_PLATFORM STREQUAL amd)
-  rocm_package_add_dependencies(DEPENDS "rocblas >= 4.2.0" "rocsolver >= 3.26.0")
+  set(rocblas_minimum 4.2.0)
+  set(rocsolver_minimum 3.27.0)
```

Comment:
Please confirm solver bump to 3.27 for 6.2 as this is different.  This is bad if solver is incorrect in release-staging and topic

Reply:
Question answered